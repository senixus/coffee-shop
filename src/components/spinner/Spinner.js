import React from "react";
import "./spinner.scss";

const Spinner = () => {
  return (
    <div class="spinner-box">
      <div class="solar-system">
        <div class="earth-orbit orbit">
          <div class="planet earth"></div>
          <div class="venus-orbit orbit">
            <div class="planet venus"></div>
            <div class="mercury-orbit orbit">
              <div class="planet mercury"></div>
              <div class="sun"></div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Spinner;
