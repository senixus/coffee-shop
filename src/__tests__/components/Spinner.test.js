import { shallow } from "enzyme";
import React from "react";
import Spinner from "../../components/spinner/Spinner";

it("expect to render Spinner component", () => {
  const wrapper = shallow(<Spinner />);
  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
