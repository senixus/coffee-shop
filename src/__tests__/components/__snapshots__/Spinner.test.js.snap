// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Spinner component 1`] = `
<div
  className="spinner-box"
>
  <div
    className="solar-system"
  >
    <div
      className="earth-orbit orbit"
    >
      <div
        className="planet earth"
      />
      <div
        className="venus-orbit orbit"
      >
        <div
          className="planet venus"
        />
        <div
          className="mercury-orbit orbit"
        >
          <div
            className="planet mercury"
          />
          <div
            className="sun"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
